app/views/boards/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<body>
+	<div class="board">
+ 		<table class="one">
+ 			<% 4.times do %>
+ 			<tr>
+ 		 		<td class="black"></td>
```

Comment:
Hahah you're gonna hate me- but check out CSS nth-child operator, which will help reduce repetition further https://davidwalsh.name/css-tables-css3-alternate-row-colors

Something like this on the rows
```
<tr class="even">
...
<tr class="odd">
```

Then in Css, something like
```
.board tr.even td:nth-child(even) { background-color: #000; }
.board tr.even td:nth-child(odd) { background-color: #FFF; }
```

Orrrrrrr if you wanted to solve this in Ruby- you could use modulo, which will allow you to determine even/odd

```
>>> 5%2
1
>>> 6%2
0
```

General pointer- if you are ever copy/pasting, solve it with code simplification instead ;)